### ISSUE TITLE
temporary terminal dialog box

### ISSUE THREAD

**Issue by haraldschilly** [sagemath/cloud#38]
core goal: _There shouldn't be the need to be create a terminal file for just a few commands._

Iteration 1: Next to the wrench icon, add a small icon for a persistent project specific terminal :-1: 

Iteration 2: 

Enhance the small single-line textbox in the file browser:
1. If the command produces some stdout/stderr output, open up a modal dialog box and display the result in a terminal. This terminal is already "dead", i.e. there is no prompt. Clicking `Esc` or `Return` just closes it and that's it. Most importantly, I can read through the text conveniently instead of that small crappy text box.
2. Additionally, add a "terminal" icon next to it, which opens up the same modal dialog with a temporary terminal in it. If there is a command in that small box, it is executed like usually in the current directory and the terminal gives you a prompt. Also, `Shift`+`Enter` instead of just `Enter` could open up that temporary terminal box dialog. `Ctrl`+`d` and similar close the prompt and also the terminal.

Dialog Box:
- displays the output of the command in a temporary terminal.
- a button "terminate" to kill it when the process doesn't stop. (for those who do not know about `Ctrl-c`.
- and also has a "close" button (`x` at top right?), which destroys the terminal and of course, closes the window. (additionally to e.g. `Esc`)

In summary, that wouldn't change much on the interface and one doesn't have the need to pile up terminal files for small commands. Also, #39 would work with that, too ... just another way to open this.
